### PR TITLE
feat: move url-webdemo to packages/

### PR DESCRIPTION
**Pull request by AhyoungRyu**
Addresses https://sendbird.atlassian.net/browse/AC-292

#### What was the issue? 
We uses this chat-ai-widget in ttps://github.com/sendbird/ai-bot-url-webdemo as its dependency. So whenever we encounter any issues in this web demo, I had to make fix and port it to the web-demo repo again. 
I thought they(chat-ai-widget & web demo repos) don't need to be managed in different repositories since they are fairly co-related. Also, it's be better to manage them as a monorepo so they can be released together when it's necessary.

#### How I tried to solve it? 
I've just moved (almost) everything from https://github.com/sendbird/ai-bot-url-webdemo to `packages/url-webdemo`. 
It would be better to manage this with Lerna for a more organized monorepo structure, but I didn't complete that in this PR. I've left it for the next step for now for the easier PR review.

Additionally, I've set up [Netlify](https://app.netlify.com/sites/chat-ai-widget/overview) and configured the base path as `packages/url-webdemo`.

### CODE REVIEW

**Comment by AhyoungRyu** [comment]
Finally made it work. For the working example with the Netlify preview see,
- widget : https://deploy-preview-42--chat-ai-widget.netlify.app/?hashed_key=ee54c144453b618f8d5f7a907230657f535d5c78cfd3591c1cc876826b8873a1&is_widget=true
- web-demo : https://deploy-preview-42--chat-ai-widget.netlify.app/?hashed_key=f58aa9880d42652979ad0f040ff208386eb7d8f53978b8652da348d2395de95b

File: packages/url-webdemo/README.md
Comment (AhyoungRyu) [inline packages/url-webdemo/README.md:1]: I made some modifications to this README to update the URLs for each environment. Previously, we had two different Netlify settings for production and preproduction, which were as follows:
- prod: https://app.netlify.com/sites/sendbird-chat-ai-bot-demo/deploys
- preprod: https://app.netlify.com/sites/sendbird-chat-ai-bot-demo-preprod/deploys

However, this time, I've merged them into a single config: https://app.netlify.com/sites/chat-ai-widget/overview. Instead, I've made a distinction based on branches: `main` <-> `develop`.
- prod deployments: based on the `main` branch and occur only when manual branch merges happen
  -  new url: https://chat-ai-widget.netlify.app/?hashed_key=${token}
- preprod deploument:  based on the `develop` branch, which serves as the base branch for this repository.
  - new url: https://develop--chat-ai-widget.netlify.app/?hashed_key=${token} 

Screenshot 2023-08-21 at 10 00 21 PM

Comment (chohongm) [reply packages/url-webdemo/README.md:1]: nice job!

**Review by chohongm — APPROVED** [review]
LGTM

**Pull request merged**